images/sqlx-cli/Dockerfile

```diff
@@ -1,22 +1,21 @@
 ################################################################################
 # Builder
 ################################################################################
-FROM rust:1.78-alpine AS builder
+FROM rust:1.85-alpine AS builder
 
 ARG VERSION
 
 ################################################################################
 
-# cargo-binstall: preparation
+# sqlx: preparation
 RUN apk add --no-cache \
-    curl \
-    bash
+    gcc \
+    musl-dev
 
-# cargo-binstall: install
-RUN curl -L --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/cargo-bins/cargo-binstall/main/install-from-binstall-release.sh | bash
-
-# sqlx: install
-RUN cargo binstall sqlx-cli@$VERSION -y
+# sqlx: build
+RUN cargo install sqlx-cli@$VERSION  \
```

Comment:
Comment for readers:

The sqlx developers do not self-publish `sqlx-cli` builds (in GitHub), so `cargo binstall` starts using `cargo-quickinstall` as fallback 3-rd party source of built artifacts.

Unfortunately, there are intermittent problems (https://github.com/cargo-bins/cargo-quickinstall/issues/268#issuecomment-2830279798), so I decided to use the explicit build (via `cargo install`).

Reply:
Hello, latest cargo-binstall have already stopped using this and switched to fly.io which shouldn't have this problem.

So upgrading cargo-binstall should work